frontend/src/app/app.component.ts

```diff
@@ -6,5 +6,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css'],
 })
 export class AppComponent {
-  title = 'NRM G&A';
+  title = 'Grants & Agreements';
```

Comment:
I think that there is a test for this string in particular. Will this fail? (If we are actually running those tests first?)

Reply:
Good catch!